sd-boot/pkg.yaml

```diff
@@ -21,16 +24,11 @@ steps:
         sha256: "{{ .systemd_sha256 }}"
         sha512: "{{ .systemd_sha512 }}"
     env:
-      LD_LIBRARY_PATH: "/toolchain/lib"
       SOURCE_DATE_EPOCH: {{ .BUILD_ARG_SOURCE_DATE_EPOCH }}
+  - network: default
     prepare:
       - |
-        tar -xzf systemd.tar.gz --strip-components=1
-
-        ln -s /toolchain/bin/echo /toolchain/bin/getent
```

Comment:
Actually there was a stub getent I didn't notice when debugging as I removed the whole block of ln's etc. Maybe we should restore it as it was

Reply:
if you provide proper `getent`, let's drop the symlink